types/src/database_sql.rs

```diff
@@ -2471,6 +2496,40 @@ impl Loader<PageId> for Database {
     }
 }
 
+#[async_trait]
+impl Loader<LanguagesForDocument> for Database {
+    type Value = Vec<Language>;
+    type Error = Arc<sqlx::Error>;
+
+    async fn load(
+        &self,
+        keys: &[LanguagesForDocument],
+    ) -> Result<HashMap<LanguagesForDocument, Self::Value>, Self::Error> {
+        let mut results = HashMap::new();
+        let document_ids: Vec<_> = keys.iter().map(|k| k.0).collect();
+
+        let rows = query_file!("queries/many_languages_for_documents.sql", &document_ids)
```

Comment:
is this loader supposed to load all languages for one document or is it supposed to do something different?